docs/CHANGELOG.rst

```diff
@@ -0,0 +1,3 @@
+CHANGELOG
+---------
```

Comment:
there's already a header in the file

Reply:
I think it was complaining at me but perhaps the suggestion below will sort it out.